tests/test-team/template-mgmt-component-tests/template-mgmt-message-template-page.component.ts

```diff
@@ -0,0 +1,123 @@
+import { test, expect } from '@playwright/test';
+import { MessageTemplatePage } from '../pages/template-mgmt-message-template-page';
+import {
+  assertFooterLinks,
+  assertGoBackLinkNotPresent,
+  assertLoginLink,
+  assertNotifyBannerLink,
+  assertSkipToMainContent,
+} from './template-mgmt-common.steps';
+import { TemplateFactory } from '../helpers/template-factory';
+import { TemplateStatus, TemplateType } from '../helpers/types';
+import { TemplateStorageHelper } from '../helpers/template-storage-helper';
+
+const templates = {
+  email: TemplateFactory.create({
+    id: 'valid-email-template',
+    version: 1,
+    name: 'test-template-cat-email',
+    message: 'test example message',
+    subject: 'test example subject',
+    templateType: TemplateType.EMAIL,
+    templateStatus: TemplateStatus.SUBMITTED,
+  }),
+  'text-message': TemplateFactory.create({
+    id: 'valid-sms-template',
```

Comment:
there are already templates in the automation tests with these ids - it's risky having multiple tests creating and deleting the same templates, can we use unique ids? just adding '-manage-template-page' to the end of these or something would work

Reply:
Swapped for uuids.

Reply:
@chris-elliott-nhsd are you happy with the change?

Reply:
Although, you suggest uuids so I'm assuming you are :-)